static/js/views/discovered-addon.js

```diff
@@ -123,6 +129,55 @@ class DiscoveredAddon {
         controlDiv.innerHTML = el;
       });
   }
+  
+  /**
+   * Handle a click on the license button.
+   */
+  handleLicense(e) {
+		if(e.target.getAttribute('data-id')) {
```

Comment:
You can use `e.dataset.id`. Ditto for every other usage below.

Reply:
nice, that will make things sleeker.